### PR TITLE
Fix server#photo not displaying on form and show

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

File: app/views/servers/show.html.erb [hunk 1]
```diff
@@ -214,7 +214,9 @@
               <%= Server.human_attribute_name(:photo) %>
             <% end %>
 
-            <%= image_tag @server.photo.variant(geometry: "x220"), class: "server_photo ms-0 mb-3" %>
+            <%= link_to @server.photo, rel: :noopener, target: :_blank do %>
+              <%= image_tag @server.photo, class: "w-100" %>
```
Comment (nicolas-brousse) [inline app/views/servers/show.html.erb]: keep variant use. Could be a bigger one than `x220`. But may be nice to still have one.

File: app/views/servers/show.html.erb [hunk 2]
```diff
@@ -214,7 +214,9 @@
               <%= Server.human_attribute_name(:photo) %>
             <% end %>
 
-            <%= image_tag @server.photo.variant(geometry: "x220"), class: "server_photo ms-0 mb-3" %>
```
Comment (nicolas-brousse) [inline app/views/servers/show.html.erb:217]: Does `.server_photo` present on css files and still used?

Otherwise could be nice to remove it.

Comment (B-Rass) [reply app/views/servers/show.html.erb:217]: Nope that's why I removed it, it's not used anywhere

Comment (nicolas-brousse) [reply app/views/servers/show.html.erb:217]: And it has already been removed from any css files?

Comment (B-Rass) [reply app/views/servers/show.html.erb:217]: It looks like so. I couldn't find any occurence of `server_photo` on all the project files

**Review by nicolas-brousse — APPROVED** [review]

**Pull request merged**